### PR TITLE
fix(tabs): added default border to secondary, cleaned up modifiers

### LINKED ISSUE #5128: Tabs - add border to secondary, clean up modifiers
react issue - https://github.com/patternfly/patternfly-react/issues/8104

Primary and secondary should both have a border by default, and should use the same modifier to remove the border.

### PR DESCRIPTION
**Pull request by jenny-s51**
Closes #5128

### CODE REVIEW

**Review by mcoker — CHANGES_REQUESTED** [review]
Looks great, thanks @jenny-s51! Just one thing - the tabs demos should currently reflect the correct usage of secondary tabs borders, and the secondary tabs in the [modal tab demo](https://patternfly-pr-5171.surge.sh/components/tabs/html-demos/modal-tabs/) do not have a bottom border. With this update, we'll need to add `tabs--HasNoBorderBottom` here, and I think we could use `tabs--IsSecondary` in favor of `pf-m-secondary`, too - https://github.com/patternfly/patternfly/blob/f060ea960c1d7dbfbee56505af4f84731a54c403/src/patternfly/demos/Tabs/examples/Tabs.md#L388

File: src/patternfly/demos/Tabs/examples/Tabs.md [hunk 1]
```diff
@@ -385,7 +385,7 @@ section: components
       {{#> modal-box-body modal-box-body--attribute=(concat 'id="' modal-template--id '-modal-description"')}}
         {{#> grid grid--modifier="pf-m-gutter"}}
           {{#> grid-item}}
-            {{#> tabs tabs--id=(concat modal-template--id '-tabs') tabs--modifier="pf-m-inset-none pf-m-secondary"}}
+            {{#> tabs tabs--id=(concat modal-template--id '-tabs') tabs--IsSecondary="true" tabs--HasNoBorderBottom="true" tabs--modifier="pf-m-inset-none"}}
```
Comment (mcoker) [inline src/patternfly/demos/Tabs/examples/Tabs.md:388]: 🔥

**Review by mcoker — APPROVED** [review]
😎

**Comment by mcoker** [comment]
@mmenestr would you mind reviewing this and make sure the tabs examples and demos look ok? This is the breaking change to add borders to secondary tabs by default.

**Comment by mmenestr** [comment]
Looks good, just missing the border in the modal example as well!

File: src/patternfly/demos/Tabs/examples/Tabs.md (hunk 1 above)
Comment (mcoker) [inline src/patternfly/demos/Tabs/examples/Tabs.md:388]: Sorry, I'm just creating extra work 😅 You had it right the first time.

```suggestion
            {{#> tabs tabs--id=(concat modal-template--id '-tabs') tabs--IsSecondary="true" tabs--modifier="pf-m-inset-none"}}
```

**Review by mcoker — CHANGES_REQUESTED** [review]

File: src/patternfly/demos/Tabs/examples/Tabs.md (hunk 1 above)
Comment (jenny-s51) [reply src/patternfly/demos/Tabs/examples/Tabs.md]: No worries! That was a nice catch either way 😄

**Review by mcoker — APPROVED** [review]
Nice!!

**Review by mmenestr — APPROVED** [review]
Looks good!! Thanks @jenny-s51

**Pull request merged**